docs/_ext/import_projects.py

```diff
@@ -32,6 +37,44 @@ class Project:
     refresh: bool = False
 
 
+TOC_TEMPLATE = """
+```{{toctree}}
+:maxdepth: 1
+:hidden:
+{toc}
+```
+"""
+
+TABLE_TEMPLATE = """
+```{{list-table}}
+{content}
+```
+"""
+
+
+class ListTableToc(SphinxDirective):
+    """Custom directive to convert list-table into both list-table and toctree."""
+
+    has_content = True
+
+    def run(self) -> list:
+        return [self.parse()]
+
+    def parse(self):
+        """Parses the list-table and adds a toctree"""
+        toc = ""
+        for doc in re.findall(r"<(/flytesnacks/.+)>", self.block_text):
```

Comment:
This directive is not generic and only works for refs that is prefixed with `flytesnacks`. Can this be documented in `ListTableToc`'s docstring?

Reply:
I'll make this configurable

Reply:
I made `ListTableToc` more generic